.gitignore

```diff
@@ -10,3 +10,4 @@ dist
 **/coverage
 # kind
 kind.exe
+**/untracked*
```

Comment:
What is this change for?

Reply:
to be able to keep WIP files between sources, without git suggesting them for stage, they keep getting added when I just do git add test/.
It's not necessary, but is it harmful?

Reply:
Might not be harmful, but totally not expected. I would say try to get it in as a part of individual PR so it is clear such change was introduced.

Reply:
ACK